DNN Platform/Modules/ResourceManager/ResourceManager.Web/src/components/actions/dnn-action-open-file/dnn-action-open-file.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { Item } from '../../../services/ItemsClient';
+import state from "../../../store/store";
+
+@Component({
+  tag: 'dnn-action-open-file',
+  styleUrl: '../dnn-action.scss',
+  shadow: true,
+})
+export class DnnActionOpenFile {
+
+  @Prop() item!: Item;
+
+  constructor(){
+  }
```

Comment:
If the constructor is not doing anything, it can be removed.

Reply:
Thanks for the tip